app/screens/search/search.js

```diff
@@ -92,7 +92,7 @@ class Search extends Component {
     componentDidUpdate() {
         const {searchingStatus, recent} = this.props;
         const recentLenght = recent.length;
-        const shouldScroll = searchingStatus === RequestStatus.SUCCESS || searchingStatus === RequestStatus.STARTED;
+        const shouldScroll = (searchingStatus === RequestStatus.SUCCESS || searchingStatus === RequestStatus.STARTED) && !this.recentRemoved;
```

Comment:
Oh that is great!! I didn't know that,.. will make the change.. awesome HH thanks